server/routeHelpers.js

```diff
@@ -57,7 +70,7 @@ module.exports = {
       }
       answerString = answerString.slice(0, answerString.length - 2);
       db.answer.postMultiple(answerString).then(result => {
-        res.end();
+        res.send();
```

Comment:
Is this supposed to return something? Typically if you are not returning anything a res.end() is used.